README.md

```diff
@@ -1 +1,119 @@
-# Advanced_Building_Blocks_enumerables
\ No newline at end of file
+# Building_Blocks# bubble_sort
+
+In this project I re-made some of the most used Ruby Enumerable classes. The project is done as part of microverse main curriculum from the Ruby Section.
+
+![screenshot](Screenshot.JPG)
+
+1. my_each for each
+2. my_select for select
+3. my_all? for all?
+4. my_any? for any?
+5. my_none? for none?
+6. my_count for count
+7. my_count() for count()
+8. my_map for map
+9. my_inject for inject
+
+
+## Built With
+
+- RUBY
+
+## Prequisites
+
+Ruby installed on your local machine.
+Terminal or similar to execute the program.
+
+## Testing
+In this project the testing was done using RSpec to test all game logic methods.
+## Installation
+
+1. Install Ruby on your machine
+
+2. Download and Unzip the folder containing this project, or clone it.
+
+3. See "Usage" below to see how to use the classes
+
+4. In the terminal, navigate to the folder containing the file and run "ruby index.rb"
```

Comment:
- [ ] [Optional] I would recommend change the name of your` index.rb` to `enumerables.rb`